scripts/mkdist.js

```diff
@@ -14,43 +14,40 @@ function mkdist (opts, cb) {
 
   var dist = opts.dist
   var ver = dist.version
-  if (ver.match(/^v/)) ver = ver.substring(1)
```

Comment:
@haadcode hm, even though there is no proper versioning, using the commit as the version would be the least you can do. I mean, in the end, master/latest/nightly is just a reference to a version (which still could be a commit). Simply providing "master" as a (mutable) version doesn't feel very user friendly and to be honest, be against what we want IPFS to stand for.